apps/client/src/index.tsx

```diff
@@ -10,19 +10,19 @@ import { RouterProvider } from 'react-router-dom';
 import { DEFAULT_LOCALE } from './constants';
 import { router } from './router';
 import { queryClient } from './data/query-client';
-import type { GlobalThis } from 'type-fest';
+import { setServiceUrl } from './services';
+import { GlobalWithResources } from './types/global-with-resources';
```

Comment:
use type import is more suggested. 
```
import { GlobalWithResources } from './types/global-with-resources
```
ref: 
- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export
- https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how

also, some eslint rule on this may also help enforce this. [@typescript-eslint/consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports) and [@typescript-eslint/consistent-type-exports](https://typescript-eslint.io/rules/consistent-type-exports)

Reply:
updating the line to `import type { GlobalWithResources } from './types/global-with-resources';`